### PR TITLE
add badges to hero section banner

### PR DESCRIPTION
**Pull request by Majid-pkz**

### CODE REVIEW

File: src/app/components/badges/BadgeDisplay.tsx [hunk 1]
```diff
@@ -1,17 +1,20 @@
...
-      <Image src={bg} alt='Background' />
-
-      <div className={styles.badgeOverlay}>
+    <div className={styles.badgeWrapper}>
+      <div className={styles.gold}>
         <Image src={goldBadge} alt='Gold Badge' />
       </div>
+      <div className={styles.silver}>
+        <Image src={silverBadge} alt='Gold Badge' />
+        <Image src={bronzeBadge} alt='Gold Badge' />
```
Comment (bhumkarpratik) [inline src/app/components/badges/BadgeDisplay.tsx:16]: I think this can be silver and bronze

Comment (bhumkarpratik) [reply src/app/components/badges/BadgeDisplay.tsx:16]: The styles class name can also be different

**Review by bhumkarpratik — COMMENTED** [review]
I think the banner image on desktop is a bit too long. I know its the same in the designs but it carries way too more space and takes focus off teacher section.
We can chop it to just where the lower end of the slab is...

File: src/app/components/badges/BadgeDisplay.tsx (hunk 1 above)
Comment (Majid-pkz) [reply src/app/components/badges/BadgeDisplay.tsx:16]: > The styles class name can also be different

Not sure what do you mean by this.  I changed class Name silver to silverAndBronze reflecting the above comment

**Review by bhumkarpratik — APPROVED** [review]
LGTM

**Pull request merged**